searx/engines/dict.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+"""
+ DICT protocol
+ @website     https://tools.ietf.org/html/rfc2229
+ @provide-api yes
+ @using-api   yes
+ @results     DICT
+ @stable      yes
+ @parse       title, content
+"""
+
+from dictionary_client import DictionaryClient
+
+categories = ['general']
+paging = False
+db = '*'
```

Comment:
According to https://pypi.org/project/py-dict-client/ db can be, for example, `'fra-eng'` ?

If yes, there are two existing dictionary engines:
* https://github.com/searx/searx/blob/master/searx/engines/dictzone.py
* https://github.com/searx/searx/blob/master/searx/engines/translated.py

There are 'activated' when the query looks like `[lang1]-[lang2] [query]`, for example `en-fr house` (in the general category).

Do you think something similar can be implemented ? Probably using a mapping between the searx languages and the dict languages ?

Reply:
DICT is not limited to multilingual dictionaries, so this couldn't cover every case. Freedict.org dictionaries packaged with Debian follows `fd-[lang1]-[lang2]` pattern, when Mueller's English-Russian dictionary packaged with Debian named `mueller7`, and it could be different with different packaging. This should probably be specified in settings.yml, e.g. if this a multilingual dict, so it could be used as it.